src/IO/MultiscaleSpatialImage.js

```diff
@@ -224,7 +224,8 @@ class MultiscaleSpatialImage {
       const dim = this.spatialDims[index]
       if (info.coords.has(dim)) {
         const coords = await info.coords.get(dim)
-        spacing[index] = coords[1] - coords[0]
+        // if has one entry for dim, just use, else find difference between first 2
+        spacing[index] = coords.length === 1 ? coords[0] : coords[1] - coords[0]
```

Comment:
`coords[0]` is the origin -- to be consistent with other behavior we should use `1.0` as the default if there is not spacing.

Reply:
Fixed.